DEVEL.md

```diff
@@ -209,14 +209,26 @@ $ git clone https://github.com/<GitHub Username>/community-operators
 
 Generate package manifests with:
 ```
-$ make packagemanifests OPERATOR_VERSION=0.X.Y
+$ make bundle OPERATOR_VERSION=0.X.Y
+```
+
+Build bundle image and push to a local docker registry:
+```bash
+$ cd community-operators/operators/intel-device-plugins-operator/0.X.Y/ && docker build . -t <Local IP Address>:5000/intel-device-plugins-operator:0.X.Y
+$ docker run -d -p 5000:5000 --restart=always --name registry registry:2
+$ cat /etc/docker/daemon.json # Add (or create) the following line to /etc/docker/daemon.json
+{
+  "insecure-registries" : ["<Local IP Address>:5000"]
+}
+$ systemctl restart docker
+$ docker push <Local IP Address>:5000/intel-device-plugins-operator:0.X.Y
 ```
 
 Verify the operator deployment works OK via OLM in your development cluster:
 ```
 $ operator-sdk olm install
 $ kubectl create namespace testoperator
-$ operator-sdk run packagemanifests community-operators/operators/intel-device-plugins-operator/ --namespace testoperator --version 0.X.Y
+$ operator-sdk run bundle <Local IP Address>:5000/intel-device-plugins-operator:0.X.Y/ -n testoperator --use-http
```

Comment:
I can see now why `localhost` fails: that's not available in the pod that pulls the bundle.

Reply:
Yeap.. It's what I said.!
But, as I told, the ip address in the local network works.

Reply:
you said localhost fails but did not explain *why*. 

>  the ip address in the local network works

this is not the requirement though. any registry available for the cluster should work and now the instructions give the impression that you need to have a single node cluster that also serves as the registry.

Reply:
> you said localhost fails but did not explain why.

Umm, I said many times "a parameter 'localhost' is delivered as it is to a container that runs commands with the parameter as the registry ip" so I even gave the [link](https://github.com/operator-framework/operator-sdk/issues/4815#issuecomment-1097276912) that has the results of "kubectl logs" and "kubectl describe pod"...

But, maybe my explation was not understandable...

Anyway, it's not an important matter.

Reply:
I did not check the logs.

> Anyway, it's not an important matter.

but why do we want to instruct to run their own registry?

Reply:
Ah, well, we don't need. Let me remove just the whole part, leaving just.. that the user needs to push the image to a registry.